includes/templates/single-web-story.php

```diff
@@ -58,27 +59,30 @@
 </head>
 <body>
 <?php
-$metadata = function_exists( 'amp_get_schemaorg_metadata' ) ? amp_get_schemaorg_metadata() : [];
+
+$metadata = Story_Post_Type::get_schemaorg_metadata();
+
 if ( isset( $metadata['publisher']['logo']['url'] ) ) {
 	$publisher_logo_src = $metadata['publisher']['logo']['url'];
 } elseif ( isset( $metadata['publisher']['logo'] ) && is_string( $metadata['publisher']['logo'] ) ) {
 	$publisher_logo_src = $metadata['publisher']['logo'];
 } else {
 	$publisher_logo_src = admin_url( 'images/wordpress-logo.png' );
 }
+
 $publisher = isset( $metadata['publisher']['name'] ) ? $metadata['publisher']['name'] : get_option( 'blogname' );
 
 $meta_images = Media::get_story_meta_images();
 
+$poster_portrait = isset( $meta_images['poster-portrait'] ) ? $meta_images['poster-portrait'] : plugin_dir_url( WEBSTORIES_PLUGIN_FILE ) . 'assets/images/fallback-poster.jpg';
+
```

Comment:
Shouldn't we use `plugins_url` above ^

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/140/commits/0be1a0d045ae259de3cfdb6be143c11404f5ccb7

Reply:
fixed in https://github.com/google/web-stories-wp/pull/140/commits/0be1a0d045ae259de3cfdb6be143c11404f5ccb7